app/coffee/Repositories/Tags.coffee

```diff
@@ -13,7 +13,11 @@ module.exports = Tags =
 		db.tags.find {"user_id" : user_id}, callback
 	
 	createTag: (user_id, name, callback = (err, tag) ->) ->
-		db.tags.insert({ user_id, name, project_ids: [] }, callback)
+		db.tags.insert({ user_id, name, project_ids: [] }, (err, tag) ->
+			if err && err.code == 11000
```

Comment:
Can we do something here to make it clear what error this is that we're checking? A comment might suffice, or assigning `DUPLICATE_ERROR_CODE = 11000` or something?